test/metrics/prom-metrics-linter/Dockerfile

```
@@ -9,7 +9,7 @@ COPY . .
 RUN go mod tidy && \
     CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -v -trimpath -ldflags "-s -w" -o /bin/ .
 
-FROM --platform=linux/${TARGETARCH} gcr.io/distroless/base:latest
+FROM --platform=${TARGETOS}/${TARGETARCH} gcr.io/distroless/base:latest
```

nit: can be done replaced by TARGETPLATFORM
```suggestion
FROM --platform=${TARGETPLATFORM} gcr.io/distroless/base:latest
```

But then you'll need to add it as a new ARG, as done for `TARGETOS` and `TARGETARCH`.

Up to you. I'll approve if you don;t like to adopt this suggestion.

---

We can go with ${TARGETOS}/${TARGETARCH} — it might make things look more explicit.